docs/intents/ref/StartCall.md

```diff
@@ -6,4 +6,40 @@ hide_title: true
 ---
 # `StartCall`
 
-_TO BE COMPLETED_
\ No newline at end of file
+Initiate a call with a contact or list of contacts.
+
+## Intent Name
+
+`StartCall`
+
+## Display Name
+
+`Start a Call`
+
+## Possible Contexts
+
+* [Contact](../../context/ref/Contact)
+* [ContactList](../../context/ref/ContactList)
+
+## Example
+
+```js
+const contact = {
+    type: 'fdc3.contact',
+    name: 'Jane Doe',
+    id: {
+        email: 'jane@mail.com'
+    }
+}
+
+fdc3.raiseIntent('fdc3.StartCall', contact)
```

Comment:
I don't think these should be prefixed with `fdc3`. The decision as per the intents spec was that standardised intents belonging to fdc3 would not have a prefix.

Reply:
agreed.  updated.